packages/client/components/AnalyticsPage.tsx

```diff
@@ -128,15 +129,16 @@ const AnalyticsPage = () => {
   const {href, pathname} = location
 
   useEffect(() => {
-    const token = window.localStorage.getItem(LocalStorageKey.APP_TOKEN_KEY)
-    // no token means authentication is required & authentication handles identify on its own
-    if (!token) return
-    const email = window.localStorage.getItem(LocalStorageKey.EMAIL)
-    if (email) {
-      safeIdentify(atmosphere.viewerId, email)
-      return
-    }
     const cacheEmail = async () => {
+      const token = await getAuthCookie(window)
```

Comment:
if i had to guess, i'd say this is why it thinks I'm logged out when i refresh /pages